channel.go

```diff
@@ -43,6 +43,9 @@ var (
 
 const (
 	ephemeralHostPort = "0.0.0.0:0"
+
+	// DefaultTraceSampleRate is the default sampling rate for traces.
+	DefaultTraceSampleRate = 1.0
```

Comment:
As discussed offline, the public library will stay at 100% so the behaviour doesn't change.

Our internal wrapper libraries can set the default to 1%.